domain/schema/controller/sql/0022-model-authorized-keys.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE model_authorized_keys (
+    model_uuid TEXT NOT NULL,
+    user_public_ssh_key_id INTEGER NOT NULL,
+    FOREIGN KEY (user_public_ssh_key_id)
+    REFERENCES user_public_ssh_key (id),
+    FOREIGN KEY (model_uuid)
+    REFERENCES model (uuid)
+);
+
+CREATE UNIQUE INDEX idx_model_authorized_keys_composite
```

Comment:
Can we add a TODO to fix the trigger?

Reply:
I have messaged Simon to discuss. Will just update when we have a fix. TODO's are where things go to die and require a bunch of work to track.

Reply:
All fixed up.